pvlib/pvsystem.py

```diff
@@ -376,17 +378,29 @@ def get_irradiance(self, solar_zenith, solar_azimuth, dni, ghi, dhi,
         poa_irradiance : DataFrame or tuple of DataFrame
             Column names are: ``'poa_global', 'poa_direct', 'poa_diffuse',
             'poa_sky_diffuse', 'poa_ground_diffuse'``.
+
+        See also
+        --------
+        :py:func:`pvlib.irradiance.get_total_irradiance`
         """
         dni = self._validate_per_array(dni, system_wide=True)
         ghi = self._validate_per_array(ghi, system_wide=True)
         dhi = self._validate_per_array(dhi, system_wide=True)
+
+        try:
+            albedo = kwargs.pop('albedo')
+        except KeyError:
+            albedo = None
```

Comment:
Why not include `albedo=None` explicitly in the function signature instead of try/except and `kwargs`?